jaxb-ri/pom.xml

```diff
@@ -171,6 +171,7 @@
         <jmockit.version>1.34</jmockit.version>
         <copyright.template.file>tools/config/copyright.txt</copyright.template.file>
         <copyright.exclude.file>tools/config/copyright-exclude</copyright.exclude.file>
+        <mrjar.sourceDirectory>${project.basedir}/src/main/java-mr</mrjar.sourceDirectory>
```

Comment:
can include <netbeans.hint.jdkPlatform>JDK_9</netbeans.hint.jdkPlatform> property;
definitely should include properties holding src/target/release compilation levels otherwise changing minimum JDK level will become a pain and quite big change

Reply:
Addressed.